CenterForOpenScience/osf.io
# [ENG-349] Feature/fix pending embargoed registration types

## Purpose


Fixes typos and line break issues in embargoed registration notification emails. 

## Changes


Modified embargoed registration email templates (`pending_embargo_non_admin.html.mako`, `pending_embargo_admin.html.mako`) to fix some language, add/remove commas, and remove html line break tags (`<br>`). 

## QA Notes



Changes to the templates should be tested for both admin and non-admin accounts through creation of new embargoed registrations. It may be beneficial to view the email representation across different email clients (gmail on desktop, gmail on mobile, airmail or other 3rd party apps) to confirm that the new template is consistently displayed properly.  

I examined the representation of the emails in worker logs and the changes appear to be reflected appropriately. 

## Documentation


N/A

## Side Effects


N/A 

## Ticket


[Ticket Link](https://openscience.atlassian.net/browse/ENG-349)
